File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/RemoteRepo_ListResult.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class RemoteRepo_ListResult(
  val startIndex: Long,
  val count: Long,
  val repos: List<ReposParams>,
  val state: RemoteRepoFetchState,
)

File: agent/bindings/csharp/RemoteRepoListResult.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class RemoteRepoListResult
  {
    [JsonProperty(PropertyName = "startIndex")]
    public int StartIndex { get; set; }
    [JsonProperty(PropertyName = "count")]
    public int Count { get; set; }
    [JsonProperty(PropertyName = "repos")]
    public ReposParams[] Repos { get; set; }
    [JsonProperty(PropertyName = "state")]
    public RemoteRepoFetchState State { get; set; }
  }
}
